training/training/celeryconfig.py

```diff
@@ -0,0 +1,11 @@
+broker_url = "sqs://"  # "redis://localhost:6379"
```

Comment:
could we do a dev vs. prod logic to detect `broker_url`. ie: if you are in prod, use `broker_url` of "sqs://", but if you are in dev, use redis local host?

Reply:
do we want to make it so the dev env automatically starts redis and celery?

Reply:
ideally yes

Reply:
will resolve after finishing rest of PR, will likely require changes to dlp-cli

Reply:
this will be followup pr?

Reply:
no